packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -101,6 +114,7 @@ describe("MafsGraph", () => {
         userEvent = userEventLib.setup({
             advanceTimers: jest.advanceTimersByTime,
         });
+        setDependencies(testDependencies);
```

Comment:
I know we have two ways of doing this in Perseus, but I'd recommend doing the following instead of calling `setDependencies()` here. 

Top of file: 

```js
import * as Dependencies from "../../dependencies";
```

```suggestion
        jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
            testDependencies,
        );
```

This has one major advantage: jest will automatically undo the mock at the end of the test. Calling `setDependencies` sets the global state of Perseus and means we really should be "undoing" this set at the end of the test. If we use a `spyOn` it'll be done automatically.